examples/sync/plot_sample_rate_test.py

```diff
@@ -44,17 +45,17 @@
 
 stim = np.zeros(int(1e6) + 1)
 stim[[0, -1]] = 1.
-stim_dur = len(stim) / 24414.
 with ExperimentController('FsTest', full_screen=False, noise_db=-np.inf,
                           participant='s', session='0', output_dir=None,
                           suppress_resamp=True, check_rms=None,
                           version='dev') as ec:
     ec.identify_trial(ec_id='', ttl_id=[0])
     ec.load_buffer(stim)
-    print('Starting stimulus')
+    print('Starting stimulus.')
     ec.start_stimulus()
-    print('Stimulus started')
+    wait_dur = len(stim) / ec.fs + 1.
+    print('Stimulus started. Please wait %d seconds' % wait_dur)
```

Comment:
Add a period after seconds? May be the least consequential comment I've ever made.